### PR TITLE
Feature request: terms of use for new user

### LINKED ISSUE #5613: Feature request: terms of use for new user
When using the admin frontend for more than a few admins (e.g. lecturers that upload videos directly from the admin ui), there has to be a way to supply a terms of use form on first login. The displayed text should off course be configurable and the user would have to accept the terms to continue. 

Frontend issue: https://github.com/opencast/opencast-admin-interface/issues/296

### PR DESCRIPTION
**Pull request by pselsa**
closes #5613

### CODE REVIEW

**Comment by pselsa** [comment]
Frontend PR: https://github.com/opencast/opencast-admin-interface/pull/825

File: etc/org.opencastproject.organization-mh_default_org.cfg
Comment (JulianKniephoff) [inline etc/org.opencastproject.organization-mh_default_org.cfg]:
```suggestion
# Flag to display terms of use in admin UI
```

File: etc/ui-config/mh_default_org/admin-ui/terms.de.html
Comment (JulianKniephoff) [inline etc/ui-config/mh_default_org/admin-ui/terms.de.html]: Please add a newline at the end here.

File: etc/ui-config/mh_default_org/admin-ui/terms.en-US.html
Comment (JulianKniephoff) [inline etc/ui-config/mh_default_org/admin-ui/terms.en-US.html]: Please add a newline at the end here.

File: docs/guides/admin/docs/configuration/admin-ui/terms.md
Comment (JulianKniephoff) [inline docs/guides/admin/docs/configuration/admin-ui/terms.md]:
```suggestion
Opencast allows you to display terms of use for new users in the admin UI. The user will have to accept the terms to continue using the UI.
```

File: docs/guides/admin/docs/configuration/admin-ui/terms.md
Comment (JulianKniephoff) [inline docs/guides/admin/docs/configuration/admin-ui/terms.md]: This gives me a 404. 🤔

File: docs/guides/admin/docs/configuration/admin-ui/terms.md
Comment (JulianKniephoff) [inline docs/guides/admin/docs/configuration/admin-ui/terms.md]:
```suggestion
If no language is provided, the english version will be displayed.
```

File: docs/guides/admin/docs/releasenotes/terms.txt
Comment (JulianKniephoff) [inline docs/guides/admin/docs/releasenotes/terms.txt]:
```suggestion
With this release you can now provide terms of use to be presented to new users in the admin UI. For configuration see [the respective docs](../configuration/admin-ui/terms.md).
```

**Review by JulianKniephoff — CHANGES_REQUESTED** [review]
Little drive-by-review

**Comment by JulianKniephoff** [comment]
I'm not sure what our process is/should be in this case exactly, but it's kinda sorta a lie to add the release note before the frontend PR is merged **and** lands in a new admin UI release **and** Opencast is updated to include it. 🤔

File: docs/guides/admin/docs/configuration/admin-ui/terms.md
Comment (pselsa) [reply docs/guides/admin/docs/configuration/admin-ui/terms.md]: made it a permalink

**Comment by pselsa** [comment]
> I'm not sure what our process is/should be in this case exactly, but it's kinda sorta a lie to add the release note before the frontend PR is merged **and** lands in a new admin UI release **and** Opencast is updated to include it. 🤔

Then let's wait until the frontend PR is merged and then we see in which release the admin-ui comes, then I can rebase this PR into the corresponding release

File: etc/ui-config/mh_default_org/admin-ui/terms.en-US.html
Comment (JulianKniephoff) [inline etc/ui-config/mh_default_org/admin-ui/terms.en-US.html]:
```suggestion
<h1>Sample terms of use</h1>
```

File: docs/guides/admin/docs/configuration/admin-ui/terms.md
Comment (JulianKniephoff) [inline docs/guides/admin/docs/configuration/admin-ui/terms.md]: I would suggest using this: https://github.com/opencast/opencast-admin-interface/blob/main/src/i18n/languages.ts#L29

It's not a permalink, so it might rot by us moving the file or moving the relevant info out of that file, but at least it gets updated with newly supported and/or dropped languages.

Your previous link was dead for a reason that should never ever happen again. (Us renaming the default branch.) (Although spoiler alert: We might actually want to change it to `develop` at some point to make it more consistent with the https://github.com/opencast/opencast. 🙈)

In reality we should point to something that is not code at all, but I couldn't really find anything else myself, sadly. :(

**Review by JulianKniephoff — CHANGES_REQUESTED** [review]
Here are some more nitpicks, sorry. ^.^

Regarding the release note: We discussed in the technical meeting that those are basically the responsibility of the release managers, so that's not a problem. It might still be a good idea to wait until the frontend PR is merged, since the docs still describe a non-existent feature. 🤔

Since I took that review as well I will just go ahead and coordinate it however I see fit. ;D

File: docs/guides/admin/mkdocs.yml
Comment (JulianKniephoff) [inline docs/guides/admin/mkdocs.yml]:
```suggestion
        - Terms of Use: 'configuration/admin-ui/terms.md'
```

File: docs/guides/admin/docs/configuration/admin-ui/terms.md
Comment (JulianKniephoff) [inline docs/guides/admin/docs/configuration/admin-ui/terms.md]:
```suggestion
# Terms of Use
```

**Review by JulianKniephoff — CHANGES_REQUESTED** [review]

**Comment by gregorydlogan** [comment]
@schuettloeffel-elsa This is really close if you want to finish the last few details.

**Comment by pselsa** [comment]
sorry, was on a longer vacation, but thanks for the reminder, all done now :)

**Comment by gregorydlogan** [comment]
I'm going to go ahead and merge this.  Aside from the issue I just identified in https://github.com/opencast/opencast-admin-interface/pull/825 this works fine, and chances are very good we'll get this into the admin prior to 17.0 cut.

**Pull request merged**
